src/cli/tc.rs

```diff
@@ -54,51 +56,43 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
+  let file_name = ".tailcallrc.graphql";
+  let file_name_yml = ".graphqlrc.yml";
   let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
```

Comment:
I know this is not part of your change, but can we parse the type here as a string or the bytes to create the file?

Since we're using it only on [88](https://github.com/tailcallhq/tailcall/pull/733/files#diff-b00853da75ac44d2390ae9c7899e6b06c03a50693ada790c3ab1e2f45ca9ffd4R88), we could get the actual type that we'll use directly instead of getting a `resource::Resource` to parse it afterward.

Reply:
this will return the string as expected

```suggestion
  let tailcallrc = include_str!("examples/.tailcallrc.graphql");
```

Reply:
Wouldn't work at runtime resource_str bundles it with the binary

Reply:
```suggestion
  let tailcallrc = include_str!("../../examples/.tailcallrc.graphql");
```